backend/environment/env.social-app-backend.template

```diff
@@ -5,7 +5,7 @@
 PRIVATE_IP=
 
 # Port number for the private (backend-facing) API to bind to (default: one higner than the public port)
-PRIVATE_PORT=
+PRIVATE_PORT=3001
```

Comment:
It seems like this initial value is needed to pass CI

Reply:
Fine for now. Actually the way I solved it was to comment out the env var. It's allowed to be absent, just not empty. I'll fix it in an upcoming PR.